docs/topics/developers.md

```diff
@@ -32,6 +32,11 @@ Follow these steps when cloning the brigade repository to use an existing `GOPAT
 - `$ git clone https://github.com/Azure/brigade $(go env GOPATH)/src/github.com/Azure/brigade`
 - `$ cd $(go env GOPATH)/src/github.com/Azure/brigade`
 
+**Note**: this leaves you at the tip of **master** in the repository where active development
+is happening. Likely you want to checkout the most recent stable tag:
+
+- `$ git checkout v0.7.0`
+
```

Comment:
Can we change that last phrase to "you might prefer to checkout the most recent stable tag"? I feel like that covers @bacongobbler 's concern (and mine), but still points brig-builders to the right path.

Reply:
No problem, I've added the update.